portfolio/src/main/webapp/script.js

```diff
@@ -17,18 +17,13 @@
  */
 var upd_time = 1500;
 
+/*
+*fetches a random greeting from server and puts it to HTML
+*/
 function addRandomGreeting() {
-  const greetings = 
-            ['Hello!', 'Привет!', 'مرحبا!', 'Ciao!', 'Buna!', '你好！', 'Bonjour!', '¡Hola!'];
-
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  fetch('/random-greeting').then(response => response.text()).then((greeting) => {
+    document.getElementById('greeting-container').innerText = greeting;
+  });
 }
 
 setInterval(addRandomGreeting, upd_time);
```

Comment:
It's a shame that the greeting-container is initially empty. You could add a call to addRandomGreeting() just before this line, so it also executes instantly.

Up to you, though.